.github/workflows/tests.yml

```diff
@@ -0,0 +1,48 @@
+# This workflow will do a clean installation of node dependencies, cache/restore them, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Cypress Tests
+on:
+  pull_request:
+    types:
+      - opened
+      - synchronize
+  push:
+    branches:
+      - '**'
```

Comment:
I think on all branches might be a bit of a waste of resources. Merges to master and env/uat branches are the important ones